parsec/include/parsec/parsec_options.h.in

```diff
@@ -120,17 +120,28 @@
 #define CMAKE_PARSEC_C_FLAGS      "@CMAKE_C_FLAGS@"
 #define CMAKE_PARSEC_C_INCLUDES   "@PARSEC_C_INCLUDES@"
 
-#cmakedefine PARSEC_HAVE_HWLOC
-#cmakedefine PARSEC_HAVE_PAPI
+/**
+ * The following 3 defines are deprecated. Instead users should
+ * rely on the corresponding PARSEC_HAVE_DEV_XXX_SUPPORT
+ */
 #cmakedefine PARSEC_HAVE_CUDA
```

Comment:
remove these 3 variables they should remain at the CMake level.

Reply:
do you want to do this before we merge?